src/substrate-lib/components/txButton.tsx

```diff
@@ -15,6 +15,15 @@ function TxButton ({
   type = 'QUERY',
   attrs = null,
   disabled = false
+}: {
+  accountPair: any,
+  label: any,
+  setStatus: any,
+  color?: any,
+  style?: any,
+  type: any,
+  attrs: any,
+  disabled?: any
```

Comment:
Export it to type for example

```typescript
 type TxButtonPropsType {
  accountPair: any,
  label: any,
  setStatus: any,
  color?: any,
  style?: React.CSSProperties | undefined,
  type: any,
  attrs: any,
  disabled?: boolean | undefined
  }
```

Just for example i put some types, but need check if its compile.